docs/security/README.md

```diff
@@ -0,0 +1,215 @@
+---
+description: >-
+    NAIS is a platform for building, deploying and operating applications in a
+    secure manner. This document describes on a high level how NAIS achieves this and what is expected of those who use NAIS.
+---
+
+# Security in NAIS
+
+NAIS is a platform for building, deploying and operating applications in a
+secure manner. This document describes on a high level how NAIS achieves this
+and what is expected of those who use NAIS.
+
+## Shared responsibility
+
+NAIS is a shared responsibility between the application team and the NAIS
+platform team. The platform team is responsible for providing securing the
+underlying infrastructure and providing the tools and necessary for secure
+software development lifecycle, and the application team is responsible for
+using these tools and following the guidelines provided by the platform team.
+
+## Security Principles
+
+NAIS is built on the following security principles:
+
+1. **Defense in depth** - NAIS is built on the principle of defense in depth.
+This means that security is built into every layer of the platform, from the
+underlying infrastructure to the application code. This is achieved by using
+multiple layers of security controls, so that if one layer fails, the next layer
+will provide protection.
+
+1. **Least privilege** - NAIS is built on the principle of least privilege. This
+means that every component of the platform is given the minimum amount of
+privileges necessary to perform its function. This is achieved by using
+role-based access control (RBAC) and by using the principle of separation of
+duties.
+
+1. **Secure by default** - NAIS is built on the principle of secure by default.
+This means that every component of the platform is configured to be secure by
+default. This is achieved by using secure defaults and by using the principle of
+least privilege.
+
+1. **Secure by design** - NAIS is built on the principle of secure by design.
+This means that every component of the platform is designed to be secure by
+design. This is achieved by using secure design patterns and by using the
+principle of least privilege.
+
+## Team isolation
+
+NAIS is built on the principle of team isolation. This means that every team has
```

Comment:
This structure seems to be repeating a lot. Maybe we could make a table of all of those? 
e.g 
| Principle | Meaning | Achieved by